webapp-django/crashstats/settings/bundles.py

```diff
@@ -222,6 +228,12 @@
         ),
         'output_filename': 'js/topcrash.min.js',
     },
+    'new_topcrashers': {
```

Comment:
I don't like this name. We should change the old one from `topcrashers` to `old_topcrashers`. 
Otherwise, when you one day, remove the old top crasher, calling what's left "new" doesn't make sense.